src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -26,20 +35,35 @@ namespace orbit_object_utils {
 
 namespace {
 
+[[nodiscard]] uint64_t ComputeRelativeAddress(
+    uint64_t offset_in_section, uint16_t section, uint64_t load_bias,
+    const llvm::FixedStreamArray<llvm::object::coff_section>& section_headers) {
+  // The address in PDB files is a relative virtual address (RVA), to make the address
```

Comment:
If we are also adding the RVA of the section, this is not an RVA, it's just the offset in the section.